src/acceptance-test/java/org/zalando/nakadi/webservice/hila/StreamBatch.java

```diff
@@ -36,30 +40,54 @@ public SubscriptionCursor getCursor() {
         return unmodifiableList(events);
     }
 
+    public StreamMetadata getMetadata() {
```

Comment:
@v-stepanov @rcillo according to the code events can not be null but it is marked as @Nullable. 
Maybe we can fix this divergence?